app/src/main/java/com/example/househomey/ViewItemFragment.java

```diff
@@ -79,7 +80,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         // On edit button click, pass item to EditItemFragment
         rootView.findViewById(R.id.edit_button).setOnClickListener(v ->
-                navigateToFragmentPage(getContext(), new EditItemFragment(item))
+                navigateToFragmentPage(getContext(), new EditItemFragment(item, this))
```

Comment:
We should not be bundling this, not passing as args

Reply:
and items

Reply:
I assume you mean we should be bundling this? Yeah, EditFragment already had a constructor and items were being passed as args. I wanted to quickly test if the listener I made was working, so just used the constructor. Will change it to use bundling